common/src/model_registration/register.py

```diff
@@ -53,15 +55,20 @@ def parse_args():
     parser.add_argument(
         "--registration_details",
         type=str,
-        help="Text File into which model registration details will be written",
+        help="Json File into which model registration details will be written",
     )
     parser.add_argument(
         "--model_info_path",
         type=str,
-        help="Json file containing metadata related to the model",
+        help="Json file containing metadata related to the downloaded model",
+        default=None,
+    )
+    parser.add_argument(
+        "--model_metadata_path",
+        type=str,
+        help="Yaml file that contains tags,properties and description",
```

Comment:
```suggestion
        help="YAML file containing model metadata confirming to Model V2 contract",
```

Reply:
Done

Reply:
Is there a commit that needs to be pushed? I still see the older description.